metadata.rb

```diff
@@ -11,6 +11,7 @@
 depends 'authbind', '>= 0.1.8'
 depends 'java'
 depends 'ark'
+depends 'build-essential'
```

Comment:
`build-essentials` should not be a hard dependency. Couple of people would say that it's a security risk to install it on each server with Graylog. There are pure Ruby MongoDB clients out there. These should be used preferable.